PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -192,17 +195,11 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
 run2_miniAOD_80XLegacy.toModify(slimmedElectronsWithUserData.userFloats,
     mvaSpring16GP = cms.InputTag("electronMVAValueMapProducer:ElectronMVAEstimatorRun2Spring16GeneralPurposeV1Values"),
     mvaSpring16HZZ = cms.InputTag("electronMVAValueMapProducer:ElectronMVAEstimatorRun2Spring16HZZV1Values"),
-    mvaFall17V1Iso = None,
-    mvaFall17V1noIso = None,
-    mvaFall17V2Iso = None,
-    mvaFall17V2noIso = None,
-    eCorr = cms.InputTag("energyCorrForEle80X","eCorr")
```

Comment:
this is removed but where are ecalTrkEnergyErrPostCorrNew etc. set for run2_miniAOD_80XLegacy? Was that forgotten?

Reply:
see below, your guess was correct